riot/__init__.py

```diff
@@ -1,5 +1,7 @@
-__all__ = ["latest", "Session", "Venv"]
+__all__ = ["latest", "Session", "Task", "Venv"]
 
-from .riot import Session, Venv
+from riot.riot import Session
+from riot.task import Task
+from riot.venv import VenvSpec as Venv
```

Comment:
If the argument `venvs` of `Task` could be made of type `List[Dict[..., ...]]` then `Venv` wouldn't be required to be part of the public API.